### PR TITLE
Small Updates in Lisk Service docs - Closes #1144

### LINKED ISSUE #1144: Updates in Lisk Service docs
- Replace betanet with testnet
- Mention new config option `GENESIS_HEIGHT` 
- Update source setup guide with market service installation steps

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #1144

### CODE REVIEW

File: modules/ROOT/pages/setup/source.adoc [hunk 1]
```diff
@@ -289,16 +296,27 @@ make build-local
...
 ----
-npm start
+npm run start
 ----
 
 This will use the default configuration and connect Lisk Service to the Lisk Mainnet.
 
 To change the default configuration, check out the page xref:{url_config}[Configuration with PM2].
 
+The market service is required to be started separately:
```
Comment (nagdahimanshu) [inline modules/ROOT/pages/setup/source.adoc]: There is no need to start `market` service separately. Running `npm run start` in root dir would start all services. We can remove this section.

File: modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -2,12 +2,13 @@
 Mona Bärenfänger <mona@lightcurve.io>
 :description: Describes the general purpose, architecture and usage of Lisk Service.
 :toc:
+:idseparator: -
+:idprefix:
 :imagesdir: ../assets/images
 :page-no-previous: true
 
 :url_api_mainnet: https://service.lisk.io/api/
```
Comment (nagdahimanshu) [inline modules/ROOT/pages/index.adoc]: We have already updated domain lisk.io to lisk.com with the PR #[664](https://github.com/LiskHQ/lisk-service/pull/664). I think it should reflect here as well. We can confirm it with @MichalTuleja

File: modules/ROOT/pages/configuration/docker.adoc [hunk 3]
```diff
@@ -67,9 +71,11 @@ services:
       core:
         condition: service_started
     environment:
-      ENABLE_HTTP_API: 'http-version1,http-version1-compat,http-status'
-      ENABLE_WS_API: 'rpc,rpc-v1,blockchain'
+      ENABLE_HTTP_API: http-status,http-version
```
Comment (nagdahimanshu) [inline modules/ROOT/pages/configuration/docker.adoc]:
```suggestion
      ENABLE_HTTP_API: http-status,http-version
```
It should be:
```
ENABLE_HTTP_API=http-version1-compat,http-status,http-version2
ENABLE_WS_API=rpc,blockchain,rpc-v2
```

**Review by nagdahimanshu — CHANGES_REQUESTED** [review]
Looks good apart from some minor changes.

**Review by nagdahimanshu — DISMISSED** [review]
LGTM 👍

File: modules/ROOT/pages/index.adoc (hunk 2 above)
Comment (MichalTuleja) [reply modules/ROOT/pages/index.adoc]: 👍

File: modules/ROOT/pages/references/configuration.adoc [hunk 4]
```diff
@@ -181,7 +181,7 @@ SERVICE_MARKET_TARGET_PAIRS=LSK_BTC,LSK_EUR,BTC_CHF
 
 # API key for https://exchangeratesapi.io/
 # Required to calculate Fiat exchange rates
-EXCHANGERATESAPI_IO_API_KEY=<api_key>
+EXCHANGERATESAPI_IO_API_KEY=EXCHANGERATESAPI_IO_API_KEY
```
Comment (MichalTuleja) [inline modules/ROOT/pages/references/configuration.adoc:200]: That would be meaningful to add some indicator that key is custom and not mandatory (can be left empty).

**Review by MichalTuleja — CHANGES_REQUESTED** [review]
Looks almost perfect, some sections are missing options that were recently added to the 0.4.2.

Compare this document: https://github.com/LiskHQ/lisk-service/blob/v0.4.2/docs/config_options.md

Can you also please add the new sections such as [caching](https://github.com/LiskHQ/lisk-service/blob/v0.4.2/docs/config_options.md#caching)?

**Review by MichalTuleja — APPROVED** [review]
LGTM 👍

**Pull request merged**